scripts/devaccess/helper_scripts/redacted.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+"""
+This script will dump config files in a clean format for developer access
+"""
+import os
+import pwd
+import sys
+import yaml
+
+CONFIG_FILE = "/usr/local/bin/redacted.yaml"
```

Comment:
config files shouldn't go in /usr/local/bin/ - why do we not use /usr/local/etc?

Reply:
Because of how this directory works, it just uploads everything to /usr/local/bin

Reply:
Ugly, but if that's what everything else does then at least we're consistent even if we're not correct.